explorer/ast/static_scope.cpp

```diff
@@ -41,6 +41,18 @@ void StaticScope::MarkUsable(std::string_view name) {
   it->second.status = NameStatus::Usable;
 }
 
+auto StaticScope::ResolveHere(std::string_view name, SourceLocation source_loc,
+                              bool allow_undeclared) const
+    -> ErrorOr<ValueNodeView> {
+  CARBON_ASSIGN_OR_RETURN(std::optional<ValueNodeView> result,
+                          TryResolveHere(name, source_loc, allow_undeclared));
+  if (!result) {
+    return ProgramError(source_loc)
+           << "could not resolve '" << name << "' in this scope";
+  }
+  return *result;
+}
+
```

Comment:
The order of functions confuses me:

ResolveHere
Resolve
TryResolve
TryResolveHere

Shouldn't either ResolveHere be after Resolve (making it consistent with the TryResolve/TryResolveHere ordering) or ResolveHere between TryResolve and TryResolveHere (making it consistent with Resolve/TryResolve)?

(also in the header)